src/components/Modal/VotesList.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { navigate } from 'gatsby-plugin-intl'
+import { Modal, ModalProps } from 'decentraland-ui/dist/components/Modal/Modal'
+import Grid from "semantic-ui-react/dist/commonjs/collections/Grid/Grid"
+import Avatar from 'decentraland-gatsby/dist/components/User/Avatar'
+import { Address } from 'decentraland-ui/dist/components/Address/Address'
+import { Vote } from '../../entities/Votes/types'
+import { Header } from 'decentraland-ui/dist/components/Header/Header'
+import { Close } from 'decentraland-ui/dist/components/Close/Close'
+import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { abbreviateNumber } from '../../entities/Votes/utils'
+import locations from '../../modules/locations'
+
+import './ProposalModal.css'
+import './VotesList.css'
+
+export type VotesListModalProps = Omit<ModalProps, 'children'> & {
+  onClickAccept?: (e: React.MouseEvent<any>) => void
+  votes?: Record<string, Vote> | null,
+}
+
+export function VotesList({onClickAccept, votes, ...props }: VotesListModalProps) {
+  const l = useFormatMessage()
+
+  return <Modal {...props} size="tiny" className={TokenList.join(['ProposalModal', 'VotesList' ,props.className])} closeIcon={<Close />}>
+  <Modal.Content className="ProposalModal__Title">
+    <Header>{l('modal.votes_list.title', { votes: Object.keys(votes || {}).length })}</Header>
+  </Modal.Content>
+  <div className="VotesList_Container_Header">
+    <Grid columns='equal'>
+      <Grid.Row className="VotesList_Divider_Line">
+        <Grid.Column width={8}>
+          <div className="VotesList_Header">{l('modal.votes_list.voter')}</div>
+        </Grid.Column>
+        <Grid.Column >
+          <div className="VotesList_Header">{l('modal.votes_list.voted')}</div>
+        </Grid.Column>
+        <Grid.Column>
+          <div className="VotesList_Header">{l('modal.votes_list.vp')}</div>
+        </Grid.Column>
+      </Grid.Row>
+    </Grid>
+  </div>
+  <div className="VotesList_Container_Items">
+    <Grid columns='equal' className="VotesList_Divider">
+      {Object.entries(votes || {}).sort((a, b) => b[1].vp - a[1].vp).map(vote => {
```

Comment:
to load profiles names:
- save the resulte of `Object.entries(votes || {}).sort((a, b) => b[1].vp - a[1].vp)` using a `useMemo`
- create a new component `<VoteListItem key={key} address={key} value={value} />`
- use the `useAsynMemo` hook inside each `VoteListItem` to load the avatar info and show the name

Reply:
this could help @oliviel